core/src/plan/join.rs

```diff
@@ -467,23 +467,23 @@ mod tests {
     #[test]
     fn basic() {
         let storage = run("
-            CREATE TABLE User (
+            CREATE TABLE Player (
```

Comment:
Now that User has become a reserved word, it has been changed because it cannot be used.